app/models/agents/dropbox_file_url_agent.rb

```diff
@@ -38,21 +40,35 @@ class DropboxFileUrlAgent < Agent
           }
     MD
 
+    def default_options
+      {
+        'link_type' => 'temporary'
+      }
+    end
+
     def working?
       !recent_error_logs?
     end
 
     def receive(events)
       events.map { |e| e.payload['paths'].split(',').map(&:strip) }
-        .flatten.each { |path| create_event payload: url_for(path) }
+        .flatten.each do |path|
+          create_event payload: (options['link_type'] == 'permanent' ? permanent_url_for(path) : temporary_url_for(path))
```

Comment:
Stylistically, I might check `options` in `url_for` and just handle both there, but this is fine too.